plugins/ruby-foreman-azure-rm/debian/rules

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/make -f
+
+PLUGIN = foreman_azure_rm
+
+cache:
+	@if [ ! -d cache ]; then \
+	  echo "INFO: Downloading gems files into cache folder"; \
+	  . debian/gem.list; \
+	  mkdir cache/; \
+	  cd cache/; \
+	  for GEM_URL in $${GEMS}; do \
+	    echo "INFO: Downloading $${GEM_URL}"; \
+	    wget --no-verbose --no-clobber $${GEM_URL}; \
+	  done; \
+	fi
+
+build: cache
```

Comment:
am I reading this right, you essentially duplicated what our jenkins does in https://github.com/theforeman/foreman-packaging/blob/deb/develop/plugins/download_gems here as a make target, and it's only really executed if the `cache` folder is not present, which it should be when run on jenkins?

wonder if it would be better to call that script instead? for consistency.

Reply:
hmm, no, looking at https://ci.theforeman.org/job/packaging_build_deb_plugin/720/arch=x86,label=debian,os=bionic/consoleFull it re-ran this as you clean `cache` in `clean`.

if you don't clean it, it'd work I think.

I'd really prefer our jenkins not downloading things twice (and technically you shouldn't have network access from inside the build-root)

Reply:
Hello,

Well this was basically needed to be able to run the package creation outside Jenkins.
If think you can safely remove it but I do think it's handy to be able to test the package building locally ;)

Reply:
I know, and that's why I proposed to just remove the cleaning of the cache from the clean step below, so the `cache` target becomes essentially noop on Jenkins, while still working for you locally.